### PR TITLE
[Enhancement] Support https load url

### PR DESCRIPTION
**Pull request by xlfjcg**

### CODE REVIEW

File: src/main/java/com/starrocks/connector/flink/tools/ConnectionUtils.java [hunk 1]
```diff
@@ -35,8 +35,13 @@ public class ConnectionUtils {
     @Nullable
     public static String selectAvailableHttpHost(List<String> hostList, int connectionTimeout) {
         for (String host : hostList) {
-            String urlStr = "http://" + host;
-            if (testHttpConnection(urlStr, connectionTimeout)) {
+            if (host == null) {
```
Comment (chaoyli) [inline src/main/java/com/starrocks/connector/flink/tools/ConnectionUtils.java:38]: Change the https to http?

File: src/main/java/com/starrocks/connector/flink/manager/StarRocksStreamLoadVisitor.java [hunk 2]
```diff
@@ -183,7 +183,10 @@ private String getAvailableHost() {
         List<String> hostList = sinkOptions.getLoadUrlList();
         long tmp = pos + hostList.size();
         while (pos < tmp) {
-            String host = "http://" + hostList.get((int) (pos % hostList.size()));
+            String host = hostList.get((int) (pos % hostList.size()));
+            if (host != null && !host.startsWith("http")) {
```
Comment (banmoy) [inline src/main/java/com/starrocks/connector/flink/manager/StarRocksStreamLoadVisitor.java:187]: How about move this logic to `StarRocksSinkOptions#getLoadUrlList` to reduce code duplication? `getLoadUrlList` will return urls already prefixed with http or https.

**Comment by banmoy** [comment]
We should update the description of `StarRocksSinkOptions#LOAD_URL` to tell users they can add http/https to the host.

**Review by banmoy — COMMENTED** [review]
LGTM

**Review by banmoy — APPROVED** [review]

**Pull request merged**